eskip/string.go

```diff
@@ -14,11 +14,16 @@ type PrettyPrintInfo struct {
 }
 
 func escape(s string, chars string) string {
-
-	s = strings.Replace(s, "\\", "\\\\", -1)
```

Comment:
The escaping of backslash was lost here

Reply:
interesting finding

Reply:
I have created a https://github.com/zalando/skipper/pull/1938 to fix